molecule/config.py

```diff
@@ -47,10 +47,12 @@ def populate_instance_names(self, platform):
         :param platform: platform name to pass to ``format_instance_name`` call
         :return: None
         """
-        for instance in self.config['vagrant']['instances']:
-            instance['vm_name'] = utilities.format_instance_name(
-                instance['name'], platform,
-                self.config['vagrant']['instances'])
+
+        if 'vagrant' in self.config:
```

Comment:
I think I'd like the code to work, where init is the only command that creates it.  I dont believe there is any other use case outside of that where it would be created?